bundles/org.openhab.binding.airgradient/src/main/java/org/openhab/binding/airgradient/internal/communication/RemoteAPIController.java

```diff
@@ -96,6 +99,31 @@ public List<Measure> getMeasures() throws AirGradientCommunicationException {
         return Collections.emptyList();
     }
 
+    public @Nullable LocalConfiguration getConfig() throws AirGradientCommunicationException {
+        ContentResponse response = sendRequest(
+                RESTHelper.generateRequest(httpClient, RESTHelper.generateConfigUrl(apiConfig)));
+        if (response == null) {
+            return null;
+        }
+
+        logger.debug("Got configuration with status {}: {}", response.getStatus(), response.getContentAsString());
```

Comment:
Debug is fine for meta data not for verbose content, use (or add) trace logger for content.
For reference: https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
Thank you, makes sense. I also found a similar log line from existing code and converted that as well.